packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md

```diff
@@ -1,151 +1,180 @@
+In this tutorial, you'll create an administrative account to control the actions that can be taken in the election application.
+
+Kadena accounts consist of four parts: 
+
+- An account name.
+- An account balance.
+- One or more public keys that can be used to sign transactions.
+- A predicate that specifies how many keys are required to sign transactions.
+
+For this tutorial, you'll create an administrative account with only one key and you'll use the default `keys-all` predicate.
+The `keys-all` predicate requires transactions to be signed by all keys in the account.
+Because you're creating this account with only one key, only one signature will be required to sign transactions. 
+You can use any text string of three to 256 characters for the account name.
+However, the convention for accounts that only have one key is to use the prefix `k:` followed by public key of the account.
+
+For a more general introduction to how public and private keys are used to sign transactions and in Kadena accounts, see [Beginner’s Guide to Kadena: Accounts + Keysets](/blogchain/2020/beginners-guide-to-kadena-accounts-keysets-2020-01-14).
+
+## Before you begin
+
+Before you start this tutorial, verify the following basic requirements:
+
+- You have an internet connection and a web browser installed on your local computer.
+- You have a code editor, such as [Visual Studio Code](https://code.visualstudio.com/download), access to an interactive terminal shell, and are generally familiar with using command-line programs.
+- You have cloned the [election-dapp](https://github.com/kadena-community/voting-dapp.git election-dapp) repository as described in [Prepare your workspace](/build/guides/election-dapp-tutorial/01-getting-started) and have checked out the `01-getting-started` branch.
+- You have the development network running in a Docker container as described in [Start a local blockchain](/build/guides/election-dapp-tutorial/02-running-devnet).
+- You have created a [Chainweaver account](/build/guides/election-dapp-tutorial/02-running-devnet#create-a-chainweaver-account) and are [connected to the development network](/build/guides/election-dapp-tutorial/02-running-devnet#connect-to-the-development-network) using your local host IP address and port number 8080.
+
+## Generate the administrative key pair
+
+There are many tools you can use to create keys, including the Chainweaver application you used in previous tutorial.
+In fact, if you followed that tutorial, you already have a public and private key pair that you could use as the basis for your administrative account. 
+However, for demonstration purposes in this tutorial, let's create a new key in Chainweaver and use the new key as the basis for your Kadena account.
+
+To create a new key pair and account:
+
+1. Verify the development network is currently running on your local computer.
+
+2. Open Chainweaver.
+
+3. Select **devnet** from the network list.
```

Comment:
Just checking, but how to add a `devnet` to chainweaver is already described earlier? If so , please ignore this comment :)

Reply:
this is covered in the previous tutorial, but good to point out the assumption I'm making..